### ISSUE TITLE
Get a staging branch

### ISSUE THREAD

**Issue by samuelWilliams99** [mlabs-haskell/bot-plutus-interface#66]
Set up a staging branch with protections

**Comment by samuelWilliams99** [comment]
Master is always stable with a staging branch, staging works nicely here as it gives us a nice chance to do manual tests.
Aka, before merging staging to master, we test all the examples on remote and local, so we can ensure all changes merged to master have been tested thoroughly

**Comment by gege251** [comment]
okay, yeah, I think that makes sense. Maybe in an ideal world where we don't need manual testing anymore, a staging branch would become unnecessary

**Comment by samuelWilliams99** [comment]
Precisely, and maybe once we get dedicated CI and plutip, we could automate this testing?
